services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,20 +43,46 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  // Send an event to the thermal manager queue
+  if (xQueueSend(thermalMgrQueueHandle, event, 1000) !=  pdTRUE) {
+    return ERR_CODE_QUEUE_FULL;
+  }
```

Comment:
make this 1 tick, since this is called from an ISR

Reply:
Actually, I made it so this doesn't get called from the ISR (the osHandler ISR calls its own QueueSend function). Should it still be 0?